codespell_lib/data/dictionary.txt

```diff
@@ -32690,6 +32694,8 @@ laguagues->languages
 laguange->language
 laguanges->languages
 laiter->later
+lamba->lambda, lambs,
```

Comment:
Also singular `lamb`, as well as any of `lama`, `mamba`, `samba` and even `lambada`?

Reply:
I'll add `lamb`. I couldn't find the other typos.